### PR TITLE
CNF 5456 Revision of the networking stack pinning

### PR DESCRIPTION
**Pull request by kquinn1204**
[CNF-5456]: Revision of the networking stack pinning

Version(s):
4.12 RN entry
Issue:
https://issues.redhat.com/browse/CNF-5456

Link to docs preview:
https://53716--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-rsp-workload-hints

QE review:

 QE has approved this change.

### CODE REVIEW

**Comment by kquinn1204** [comment]
peer review already carried out in https://github.com/openshift/openshift-docs/pull/53667 but redoing PR as I messed up some unrelated stuff

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -456,18 +456,28 @@ Enabling this service monitor eliminates the possibility that two queries sent a
...
 Otherwise, alert receivers that use the files will fail to deliver notifications.
 ====
 
 
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-12-rsp-workload-hints"]
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster
+At the cluster level by default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. At the container level, a crio hook script also sets an RPS mask for all virtual network devices.
```
Comment (kelbrown20) [inline release_notes/ocp-4-12-release-notes.adoc:473]: I believe we write `crio` as this format when referring the command, and generally we use the `CRI-O` formating in sentences. I can't tell if this is referencing the `crio` command, but if so, that term should be in backtics. If not than use the `CRI-O` format

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -456,18 +456,28 @@ Enabling this service monitor eliminates the possibility that two queries sent a
...
+
+With this update, if you set `spec.workloadHints.realTime` in the performance profile to `False`, the system also disables both the systemd service and the crio hook script which set the RPS mask. The system disables these RPS functions because RPS is typically relevant to use cases requiring low-latency, realtime workloads only.
```
Comment (kelbrown20) [inline release_notes/ocp-4-12-release-notes.adoc:475]: Same thing with the `crio`

**Review by kelbrown20 — COMMENTED** [review]
Great work! One little question/nit, otherwise this LGTM!

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-12-release-notes.adoc:473]: checked with @rohennes (original writer) and we will go with `CRI-O`

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (kquinn1204) [reply release_notes/ocp-4-12-release-notes.adoc:475]: going to go with what you say.

**Pull request merged**